.github/workflows/test_open_data_assets.yaml

```diff
@@ -0,0 +1,58 @@
+name: test-open-data-assets
+
+on:
+  schedule:
+    # Every third day of the month at 9am CST (3pm UTC). This schedule
+    # makes sense since most of our assets are configured to refresh on the
+    # first day of the month. Some assets have a biweekly schedule such
+    # that they also refresh in the middle of the month, but we ignore that
+    # in order to make scheduling simpler.
+    - cron: '0 15 3 * *'
+
+jobs:
+  test-open-data-assets:
+    runs-on: ubuntu-latest
+    # These permissions are needed to interact with GitHub's OIDC Token endpoint
+    # so that we can authenticate with AWS
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
**nitpick:** I think we bumped to `checkout@v4` for other repos.

Reply:
Oops, good catch! Done in [cbba969](https://github.com/ccao-data/data-architecture/pull/156/commits/cbba969b8f311c9db3e08b680d51c92a387dd070).